sklearn/preprocessing/_encoders.py

```diff
@@ -210,6 +223,10 @@ class OneHotEncoder(_BaseEncoder):
         (in order of the features in X and corresponding with the output
         of ``transform``).
```

Comment:
Note that drop categories are included here (although this does make it hard to identify the correspondence between columns and categories so I'm not entirely sure that's the right design)